### DISCUSSION TITLE
Refit NuGet package update

### DISCUSSION THREAD

**Discussion by marcominerva** [reactiveui/refit#1493] [General]
The **Refit.HttpClientFactory** package on NuGet has been updated to version 6.5.1. However, the main **Refit** package is still at version 6.3.2, even if there are updates to it in the GitHub repository. When will it be released?

**Comment by ChrisPulman** [comment]
We need to check the build mechanism, all the Refit releases should be the same matching version.
Thank you for pointing this out, we will try to resolve this as soon as possible.

**Comment by 0xced** [comment]
[Refit 6.4.1](https://www.nuget.org/packages/Refit/6.4.1) and [Refit 6.5.1](https://www.nuget.org/packages/Refit/6.5.1) packages exist on NuGet but they are both [unlisted](https://learn.microsoft.com/en-us/nuget/nuget-org/policies/deleting-packages#unlisting-a-package).

Were they unlisted on purpose or by error?

**Reply by ChrisPulman** [reply]
Thank you for the feedback, we'll look into the cause. As far as I am aware they shouldn't be unlisted.

**Comment by lotz** [comment]
It looks like these packages are still unlisted on nuget.org. Will they be unlisted before 7.0.0 is released? Thanks!

**Comment by glennawatson** [comment]
Update for anyone landing here: version alignment across the Refit packages has since been addressed and releases now ship matching versions. The old 6.4.1 / 6.5.1 `Refit` packages on nuget.org were unlisted, but current and future releases (7.x and onward) are published normally and listed.

If you hit a specific version mismatch on a current release, please open a new issue with the exact package versions. Closing this as resolved.
